### PR TITLE
fix indentation issues

### PR DESCRIPTION
**Pull request by a14n**
This PR fixes indentation issues (and spotted a bug)

### CODE REVIEW

File: packages/flutter/lib/src/cupertino/switch.dart [hunk 1]
```diff
@@ -216,9 +216,10 @@ class _CupertinoSwitchState extends State<CupertinoSwitch> with TickerProviderSt
   }
 
   void _handleTapDown(TapDownDetails details) {
-    if (isInteractive)
+    if (isInteractive) {
       needsPositionAnimation = false;
       _reactionController.forward();
+    }
```
Comment (a14n) [inline packages/flutter/lib/src/cupertino/switch.dart:222]: The bug was here

Comment (Piinks) [reply packages/flutter/lib/src/cupertino/switch.dart:222]: Can you add a test that verifies this? Maybe this should be separated into another change.

**Review by Piinks — COMMENTED** [review]
Thanks for tidying! 🧹  
Also, nice find on the Cupertino switch bug.

File: packages/flutter/lib/src/cupertino/switch.dart (hunk 1 above)
Comment (a14n) [reply packages/flutter/lib/src/cupertino/switch.dart:222]: After a quick look at this code that I'm quite foreign to, I don't really know how to test it :-/

Comment (Piinks) [reply packages/flutter/lib/src/cupertino/switch.dart:222]: I can do it in a separate change if you'd like to revert. :) I just filed https://github.com/flutter/flutter/issues/84390 to track.

Comment (a14n) [reply packages/flutter/lib/src/cupertino/switch.dart]: Thanks! I just reverted the change in `cupertino/switch.dart` and rebased on master.

**Comment by a14n** [comment]
PTAL

**Review by Piinks — APPROVED** [review]
LGTM! Thank you! Reaching out to @Hixie for test exemption before this can land. :)

**Comment by Hixie** [comment]
test-exempt: reformatting with no semantic change

**Pull request merged**